app/login/page.tsx

```diff
@@ -38,18 +38,23 @@ export default function Login({
     const supabase = createClient(cookieStore)
 
     const { error } = await supabase.auth.signUp({
-      email,
-      password,
-      options: {
-        emailRedirectTo: `${origin}/auth/callback`,
-      },
-    })
-
-    if (error) {
-      return redirect('/login?message=Could not authenticate user')
-    }
+      email: email,
+      password: password,
+          });
+          if (error) {
+              console.log(error);
+              if (error == null){
+                console.log(error);
+                  
+              } else {
+                  console.log(error);
+                  // Password must be 6 letters
```

Comment:
Is this the only password requirement? I thought we'd go for something more standard and detailed like 8+ characters, lower-case, upper-case, number,  special character,

Reply:
Ahh yes, I should also add that in, will do that now.

Reply:
@LukeHosk You would have to change it from the projects auth settings which I don't seem to have, any ideas why I can't see it when I sign in with github?